.github/configs/ct.yaml

```diff
@@ -0,0 +1,6 @@
+target-branch: master
```

Comment:
As per my other comment, I think you should remove this.

Reply:
@shimoncohen see my comment above